tests/helpers/docker/tendermint.py

```diff
@@ -305,29 +341,7 @@ def _create_config(self, nb_nodes: int) -> None:
             cmd.append(f"--hostname=node{i}")
 
         subprocess.run(cmd)  # nosec
-        for config_file in Path().cwd().glob("nodes/**/*.toml"):
```

Comment:
This is an issue that our e2e tests had. We never entered this loop, because we were not detecting the config files.